tox.ini

```diff
@@ -30,4 +31,4 @@ commands =
 [testenv:flake8]
 basepython = {[test]basepython}
 deps = flake8
-commands = flake8 .
+commands = flake8 setup.py pypom
```

Comment:
If I remember well the flake8 command failed on travis because it tried to launch flake8 against non PyPOM code (virtualenv?). I'll have a look at it another time